src/borg/fuse.py

```diff
@@ -340,7 +340,7 @@ def read(self, fh, offset, size):
                     # evict fully read chunk from cache
                     del self.data_cache[id]
             else:
-                data = self.key.decrypt(id, self.repository.get(id))
+                data = self.key.decrypt(id, self.repository_uncached.get(id))
```

Comment:
The history of the file doesn't make it look to me like that was all that intentional.

Note that repeated reads of the same VFS file are caught by kernel caching. The only case where this can make a (big) difference is e.g. reading deduplicated files in alternation or something like that, where there actually would be any advantage from caching chunks.